### PR TITLE
Create Makefile

### PR DESCRIPTION
**Pull request by neoformit**
@cat-bro I'm sure you have other things to add to this

### CODE REVIEW

**Comment by cat-bro** [comment]
@neoformit this is a really good idea. It will simplify running playbooks and will make playbook automation easier.

Since there are maybe 30+ playbooks I'm wondering if the run command can be generalised, i.e. 'run-XYZ' or maybe 'run XYZ' would result in the running of 'XYZ_playbook.yml'.

**Comment by neoformit** [comment]
Good idea - I'm not too familiar with Makefiles but I'll see if there's a way to parameterize it.

**Comment by neoformit** [comment]
Standard use of `make` allows passing of named params, which might be too wordy for us:

```
make run book=dev
```

The following is more concise but is a hack and non-standard use of `make`...

```make
# If the first argument is "run"...
ifeq (run,$(firstword $(MAKECMDGOALS)))
  # get the second argument for "run"
  PLAYBOOK := $(wordlist 2, 2, $(MAKECMDGOALS))
  # ...and turn it into do-nothing target
  $(eval $(PLAYBOOK):;@:)
endif

run:
    ansible-playbook -i hosts --key-file $(KEY_FILE) $(PLAYBOOK)_playbook.yml
```

This allows running with a positional arg like:

```
make run dev
```

But it wouldn't handle mis-spelt playbook names elegantly. We'd be better off using a bash script for that.

**Comment by cat-bro** [comment]
I love the non-standard use of Make option. I just copied it into my infrastructure folder and ran `make run dev-pulsar`. It is very cool.

**Comment by neoformit** [comment]
I'll add that in :grin:

**Review by cat-bro — APPROVED** [review]
